src/test/java/picard/sam/SetNmMdAndUqTagsTest.java

```diff
@@ -67,4 +83,15 @@ private void fixFile(final File input, final File output, final File reference)
         SetNmMdAndUqTags setNmMdAndUqTags = new SetNmMdAndUqTags();
         Assert.assertEquals(setNmMdAndUqTags.instanceMain(args), 0, "Fix did not succeed");
     }
+
+    private void calcUQ(final File input, final File output, final File reference) throws IOException {
+
+        final String[] args = new String[] {
+                "INPUT="+input,
+                "OUTPUT="+output,
+                "REFERENCE_SEQUENCE="+reference };
+
+        CalculateUqTag calculateUqTag = new CalculateUqTag();
+        Assert.assertEquals(calculateUqTag.instanceMain(args), 0, "Fix did not succeed");
```

Comment:
IMO assert text like this doesn't add value. In this case the text doesn't seem to match the operation; even if it did, it doesn't tell us much about the actual failure.